ui/ui.go

```diff
@@ -258,7 +262,8 @@ func (h *uiHandler) Certificate() http.HandlerFunc {
 
 		head := headTemplate{
 			fmt.Sprintf("Certificate - %s", cert.CommonName),
-			"site.css",
+			h.mix("/css/site.css"),
+			h.mix("/js/site.js")
```

Comment:
Compilation error here, need trailing comma.

Reply:
What makes it require the trailing comma? Is it the multiline?